tests/init_test.go

```diff
@@ -32,18 +32,31 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/ethereum/go-ethereum/core/vm"
 	"github.com/ethereum/go-ethereum/params/types/ctypes"
 	"github.com/ethereum/go-ethereum/params/types/multigeth"
 )
 
 // Command line flags to configure the interpreters.
 var (
-	testEVM   = flag.String("vm.evm", "", "EVM configuration")
-	testEWASM = flag.String("vm.ewasm", "", "EWASM configuration")
+	// PTAL: What is the API of this value? => filepath<str/ing>,capabilities<k=v>,...
+	testEVM   = flag.String("evmc.evm", "", "EVMC EVM1 configuration")
+	testEWASM = flag.String("evmc.ewasm", "", "EVMC EWASM configuration")
```

Comment:
@tzdybal Maybe we should consider using `evmc6.*`, since version 7 is the latest, and we should expect the version to continue stepping.